infor-design/enterprise
# 5021/5056 - Fix multiselect dropdown list position on mobile

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR fixes the position of dropdown list (multiselect) on mobile. I also added some minor adjustment on dismissible icon that I noticed.

Added to this PR to fix the dropdown list flows up or detached to the input field if the browser is smaller.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5021
Closes https://github.com/infor-design/enterprise/issues/5056

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app

Test #1 
- Go to http://localhost:4000/components/multiselect/example-compact.html
- Open it via browserstack on mobile or the built-in device in chrome toolbar
- Click to open `Mutliselect (with tags)`
- Select multiple items
- The position of dropdown list should be stay in its place and able to select multiple items
- Check the position of the close icon
- Should be align to the text
- Test this also in classic

Test #2 
- Go to http://localhost:4000/components/dropdown/test-sizes.html
- Make the browser smaller, make the height `700px` to have scrollbar
- Open the last dropdown below
- Make a search, type `45`
- The list should not be detached or flow up

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
